src/dnssec/sign/keys/keyset.rs

```diff
@@ -386,6 +446,14 @@ impl KeySet {
             keystate.present = true;
             keystate.signer = true;
             key.timestamps.published = Some(now.clone());
+
+            // Add algorithm
```

Comment:
Nit: "add" sounds like it is making a change to something.
```suggestion
            // Remember algorithm
```